index.js

```diff
@@ -16,12 +21,29 @@ module.exports = () => {
 			options
 		});
 
+		// Shim onclick event
+		var notification;
+		setTimeout(function () {
+			var onclickOld = notification.onclick;
+			notification.onclick = function () {
+				ipc.send('notification-onclick-shim', notification);
+				if (onclickOld) onclickOld();
+			};
+		}, 1);
+
+		// Apply overrides
+		options = Object.assign({}, options);
+		if (settings.forceSilent) options.silent = true;
+		if (settings.bodyOverride) options.body = settings.bodyOverride;
```

Comment:
I'm not sure what's actually happening here. From the comment it seems that `options` should override `settings`, but the original values of `settings` is what seems to be used? Is the returned `settings` meant to be changed by whoever uses the library, and can be changed at runtime?